modules/media/ajax/FileUpload.php

```diff
@@ -187,34 +187,36 @@ function uploadFile()
 function getUploadFields()
 {
 
-    $db =& Database::singleton();
-
-    $instruments = $db->pselect(
-        "SELECT Test_name FROM test_names ORDER BY Test_name",
-        []
-    );
-    $candidates  = $db->pselect(
-        "SELECT CandID, PSCID FROM candidate ORDER BY PSCID",
-        []
-    );
-
-    $instrumentsList = toSelect($instruments, "Test_name", null);
-    $candidatesList  = toSelect($candidates, "PSCID", null);
-    $candIdList      = toSelect($candidates, "CandID", "PSCID");
-    $visitList       = Utility::getVisitList();
-    $siteList        = Utility::getSiteList(false);
-    $languageList    = Utility::getLanguageList();
+    $db   =& Database::singleton();
```

Comment:
i think going forward we're using \NDB_Factory::singleton()->database() and namespacing